src/input/controller.js

```diff
@@ -202,19 +202,11 @@ class Controller {
             throw new Error('Invalid button');
         }
         // Mouse actions are stored as negative numbers to prevent clashing with keycodes.
-        if (this._actions[action]) {
-            this._actions[action].push({
-                type: ACTION_GAMEPAD,
-                button: button,
-                pad: pad
-            });
-        } else {
-            this._actions[action] = [{
-                type: ACTION_GAMEPAD,
-                button: button,
-                pad: pad
-            }];
-        }
+        this.appendAction(action, {
```

Comment:
Would you mind also updating the comment above this line which is clearly wrong?

Reply:
I have updated the pull request with this fix